assets/js/scale.fix.js

```diff
@@ -0,0 +1,20 @@
+fixScale = function(doc) {
```

Comment:
I think this file is automatically downloaded by Jekyll and should be included in the repo. It's not in our internal docs repo.

Reply:
Same: you are corred, @David-N-Perkins. File now removed.